test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,181 @@
 
 describe("Plots", () => {
   describe("AreaPlot", () => {
-    // HACKHACK #1798: beforeEach being used below
-    it("renders correctly with no data", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
-      let plot = new Plottable.Plots.Area();
-      plot.x((d: any) => d.x, xScale);
-      plot.y((d) => d.y, yScale);
-      assert.doesNotThrow(() => plot.renderTo(svg), Error);
-      assert.strictEqual(plot.width(), 400, "was allocated width");
-      assert.strictEqual(plot.height(), 400, "was allocated height");
-      svg.remove();
-    });
+    describe("Basic Usage", () => {
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
```

Comment:
Do we need these 2 variables?  I personally don't really see the need to assert the width/height of the plot.  Do you think it's necessary?